app/views/substitutions/_form.html.haml

```diff
@@ -1,6 +1,5 @@
 = simple_form_for @substitution do |form|
   /= form.input :closed, disabled: false
-  = form.association :mentor 
```

Comment:
New action doesn't have something like .new_record?. Isn't _form.html.haml used only by new? Index and show have their own views. I don't see any reference to the _form.html.haml there.